library/modules/bigip_snat_pool.py

```diff
@@ -27,6 +27,14 @@
     type: list
     aliases:
       - member
+  description:
+    description:
+      - A general description of the SNAT pool, provided by the user for their
+        benefit. It is optional.
+    type: str
+    default: ""
+    aliases:
+      - desc
```

Comment:
@wojtek0806 does it make sense to allow 'desc' as an alias for the description field?

Reply:
Removed in the next code push.